### ISSUE TITLE
About the result of directly pretrained on target fine-grained datasets.

### ISSUE THREAD

**Issue by fankewen** [sungnyun/openssl-simcore#2]
Hi, I have read your paper and that is a great work. But I faced some problems when pretraining on fine-grained datasets. I chose BYOL with resnet50 and Cars datasets. I set batchsize=2048, epoch=1000, cosine lr schedule with lr=1e-1, and weight decay=1e-4. The loss has decayed to 0.1 but the linear evaluation (fixed backbone) result is far away from 50%. I am confused and hope to know what're suitable settings, or have I missed the experiments' details?

**Comment by raymin0223** [comment]
Hi, thank you for your interest in our paper!

We assume that the reason for the different results may be due to the different hyperparameter settings.
Specifically, we found that BYOL is highly sensitive to hyperparameters, particularly the momentum value of the EMA update.
Thus, we conducted a large number of ablation studies and found that the following settings produced the best results for pretraining on Cars dataset ($X$):

- Batch size of 512,
- 1000 epochs,
- Adam optimizer with a learning rate of 1e-3 and weight decay of 1e-6,
- Cosine scheduler,
- EMA momentum of 0.996 **without scheduling (fixed for all epochs)**,
- DataParallel of 4 GPUs

Note that we used this model as a retrieval model of SimCore, and EMA momentum has scheduled from 0.996 to 1.0 for SimCore.

For other SSL pretraining methods, you can refer to Appendix B.2 for the hyperparameters we used.
However, since we could have missed some details, please contact us if you have some issues or require further clarification.

**Comment by fankewen** [comment]
Thank you for your kindness! I'll have a try.
